web/ui/module/codemirror-promql/src/complete/hybrid.ts

```diff
@@ -198,6 +204,7 @@ export function computeStartCompletePosition(state: EditorState, node: SyntaxNod
     // So we have to analyze the string about the current node to see if the duration unit is already present or not.
     (node.type.id === NumberDurationLiteralInDurationContext && !durationTerms.map((v) => v.label).includes(currentText[currentText.length - 1])) ||
     (node.type.id === NumberDurationLiteral && node.parent?.type.id === 0 && node.parent.parent?.type.id === SubqueryExpr) ||
+    (node.type.id === FunctionCallBody && isItATopKLikeAggregationFunc(node) && node.firstChild !== null) ||
```

Comment:
Nice! I had to laugh a bit at the function name though 😅 How about `isAggregatorWithParam()`, similar to the naming in https://github.com/prometheus/prometheus/blob/b6107cc888f254cc12d3cafffc5454d483097cd5/web/ui/mantine-ui/src/promql/utils.ts#L196-L203

Reply:
lol indeed, much better. Thanks !